helm-chart/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: kube-downscaler
+apiVersion: v1
+version: 0.1.0
+appVersion: 0.5.1
+description:  A Helm chart for kube-downscaler
+keywords:
+- k8s pods scheduler
+- scheduled lifecycle
+home: https://www.opengov.com
```

Comment:
Please note that I don't want to use this repo as advertisement space for whatever company/org, so I would prefer to have a more natural "home" link (e.g. URL of this repo). I'm fine with putting your org as contributor/acknowledgement in the README.